LegitimateURLShortener.txt

```diff
@@ -2166,6 +2166,10 @@ $removeparam=referer,domain=bodykind.com
 ! https://analytics.bluekai.com/site/16231?phint=event=click&phint=campaign=BRAND-TAB&phint=platform=search&done=https://example.com/
 ||analytics.bluekai.com^$removeparam=phint
 
+! https://www.hao123.com/?tn=93288632_hao_pg
+! It is the rebate parameter for the page.
```

Comment:
Same as my other comment. I think this should link to this PR (both for record-keeping purposes and to give you credit)

Reply:
OK, I'll add it next time.